ding/envs/env/ding_env_wrapper.py

```diff
@@ -127,16 +126,17 @@ def random_action(self) -> np.ndarray:
         return random_action
 
     def _wrap_env(self) -> None:
-        wrapper_cfgs = self._cfg.get('env_wrapper', None)
-        if wrapper_cfgs is None:
-            wrapper_cfgs = 'default'
+        # wrapper_cfgs: Union[str, List]
+        wrapper_cfgs = self._cfg.get('env_wrapper', 'default')
         if isinstance(wrapper_cfgs, str):
             wrapper_cfgs = get_default_wrappers(wrapper_cfgs, self._cfg.get('env_id', None))
         self._wrapper_cfgs = wrapper_cfgs
-        for wrapper_cfg in self._wrapper_cfgs:
-            if wrapper_cfg.get('disable', False):
-                continue
-            self._env = create_env_wrapper(self._env, wrapper_cfg)
+        for wrapper in self._wrapper_cfgs:
```

Comment:
If "wrapper" is from "self._wrapper_cfgs", which is from "wrapper_cfgs", which is from "get_default_wrappers" or "self._cfg.get('env_wrapper', 'default')", both of which would return list of string.
So what is the cases when wrapper is callable?